packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.vue

```diff
@@ -1,19 +1,25 @@
 <template>
-  <SfInput
-    type="number"
-    :value="qty"
-    :aria-label="ariaLabel"
-    :disabled="disabled"
-    class="sf-quantity-selector"
-    @input="$emit('input', parseInt($event, 10))"
-  />
+  <div class="sf-quantity-selector">
+    <SfButton class="sf-quantity-selector__button">-</SfButton>
```

Comment:
@pspaczek @AdamPawlinski wdyt to use here SfIcon instead of hardcoded sign ?

Reply:
@martaradziszewska IMO is just a text :)